.github/workflows/test_on_push.yaml

```diff
@@ -0,0 +1,23 @@
+name: Run tests on any Push event
+
+on:
+  push:
+    tags:
+      - '!refs/tags/*'
+    branches:
+      - '*'
+jobs:
+  run_tests:
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@master
+      - name: Install requirements
+        run: |
+          sudo apt install -y git gcc cmake make unzip curl
```

Comment:
No need to install majority of those tools because they are part of preinstalled software on GitHub actions runners https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md

Reply:
Thanks for your comments.

I don't find gcc and make in this list. I left only them.

Reply:
They are part of `build-essentials`